tests/test_tags.py

```diff
@@ -7,27 +10,36 @@ def test_add_tag(hlwm):
     hlwm.callstr('add foobar')
 
     assert hlwm.get_attr('tags.count') == '2'
+    assert hlwm.get_attr('tags.1.client_count') == '0'
+    assert hlwm.get_attr('tags.1.client_count') == '0'
+    assert hlwm.get_attr('tags.1.curframe_wcount') == '0'
+    assert hlwm.get_attr('tags.1.curframe_windex') == '0'
```

Comment:
I'm not 100% sure how to separate the tests for *tags*, *layouts* and *frames* (should the latter be another group of tests?). These `curframe_` attributes seem to fit into multiple topics.